index.css

```diff
@@ -31,6 +39,37 @@ header {
   left: 12px;
 }
 
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+
+  /* -webkit-transition: all 0.3s ease-in-out; */
+  transition: all 0.1s ease-in-out;
+  background-color: white;
+}
+
+.hamburger {
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.